File: core/src/main/java/com/toocol/termio/core/term/commands/processors/TestCmdProcessor.java
package com.toocol.termio.core.term.commands.processors;

import com.toocol.termio.core.term.commands.TermioCommandProcessor;
import com.toocol.termio.core.term.core.Term;
import com.toocol.termio.utilities.utils.Tuple2;
import io.vertx.core.eventbus.EventBus;

/**
 * @author ：JoeZane (joezane.cn@gmail.com)
 * @date: 2022/7/12 1:18
 * @version: 0.0.1
 */
public class TestCmdProcessor extends TermioCommandProcessor {
    @Override
    public void process(EventBus eventBus, String cmd, Tuple2<Boolean, String> resultAndMsg) {
        Term.getInstance().printTest();
    }
}

File: core/src/main/java/com/toocol/termio/core/term/commands/processors/TestCmdProcessor.kt
package com.toocol.termio.core.term.commands.processors

import com.toocol.termio.core.term.commands.TermCommandProcessor
import com.toocol.termio.core.term.core.Term
import com.toocol.termio.utilities.utils.Tuple2

/**
 * @author ：JoeZane (joezane.cn@gmail.com)
 * @date: 2022/7/12 1:18
 * @version: 0.0.1
 */
class TestCmdProcessor : TermCommandProcessor() {
    override fun process(cmd: String, resultAndMsg: Tuple2<Boolean, String?>): Any? {
        Term.printTest()
        resultAndMsg.first(true)
        return null
    }
}